test/resolver/resolver.spec.ts

```diff
@@ -89,8 +89,6 @@ describe('Resolver', () => {
 
       const error: ResolutionError<Error> = errorHandler.firstCall.firstArg;
       expect(error).to.be.instanceof(ResolutionError);
-      expect(error.cause).to.be.an('error');
-      expect(error.cause).to.have.property('message').that.equals('custom');
```

Comment:
Let's keep the assertion and add the `cause` back somehow.

Reply:
Should I just provide that es2022 TS code here then? As a kind of TS polyfill?